trin-beacon/src/jsonrpc.rs

```diff
@@ -119,6 +119,32 @@ async fn complete_request(network: Arc<BeaconNetwork>, request: BeaconJsonRpcReq
                 None => Err("Beacon client not initialized".to_string()),
             }
         }
+        BeaconEndpoint::FinalityUpdate => {
+            let beacon_client = network.beacon_client.lock().await;
+            match beacon_client.as_ref() {
+                Some(client) => {
+                    let update = client.get_finality_update().await;
+                    match update {
+                        Ok(update) => Ok(json!((update))),
+                        Err(err) => Err(err.to_string()),
+                    }
+                }
+                None => Err("Beacon client not initialized".to_string()),
+            }
+        }
+        BeaconEndpoint::OptimisticUpdate => {
+            let beacon_client = network.beacon_client.lock().await;
+            match beacon_client.as_ref() {
+                Some(client) => {
+                    let update = client.get_optimistic_update().await;
+                    match update {
+                        Ok(update) => Ok(json!((update))),
```

Comment:
```suggestion
                        Ok(update) => Ok(json!(update)),
```

isn't this extra bracket redundent?